internal/api/ui/login/static/i18n/sv.yaml

```diff
@@ -0,0 +1,500 @@
+    RequestTypeNotSupported: Request av en typ som inte stöds
+    MissingParameters: Obligatorisk parameter saknas
+  User:
+    NotFound: Användaren hittades inte
+    AlreadyExists: Användaren finns redan
+    Inactive: Användaren är inaktiverad
+    NotFoundOnOrg: Användaren finns inte i den valda organisationen
+    NotAllowedOrg: Användaren är inte medlem i den valda organisationen
+    NotMatchingUserID: Användaren och den användare som anges i authrequest stämmer inte överens
+    UserIDMissing: UserID är inte angett
+    Invalid: Felaktiga userdata
+    DomainNotAllowedAsUsername: Domänen är reserverad och kan inte användas
+    NotAllowedToLink: Användaren har inte rätt att kopplas ihop med ett externt konto
+    Profile:
+      NotFound: Profilen finns inte
+      NotChanged: Profilen har inte ändrats
+      Empty: Profilen saknar innehåll
+      FirstNameEmpty: Förnamnet i profilen är tomt
+      LastNameEmpty: Efternamnet i profilen är tomt
+      IDMissing: Profile ID saknas
+    E-Post:
+      NotFound: E-Post finns inte
+      Invalid: E-Post är felaktig
+      AlreadyVerified: E-Posten är redan bekräftad
+      NotChanged: E-Posten oförändrad
+      Empty: E-Post tomt
+      IDMissing: E-Post ID saknas
+    Phone:
+      NotFound: Mobilnr finns inte
+      Invalid: Mobilnr är ogiltigt
+      AlreadyVerified: Mobilnr är redan bekräftad
+      Empty: Mobilnr tomt
+      NotChanged: Mobilnr oförändrat
+    Address:
+      NotFound: Adress saknas
+      NotChanged: Adress oförändrad
+    Användarnamn:
+      AlreadyExists: Användarnamnet finns redan
+      Reserved: Användarnamnet är upptaget
+      Empty: Användarnamnet är tomt
+    Lösenord:
+      ConfirmationWrong: Lösenorden stämmer inte överens
+      Empty: Lösenordet är tomt
+      Invalid: Lösenordet är ogiltigt
+      InvalidAndLocked: Lösenordet är ogiltigt och användaren är spärrad. Ta kontakt med systemansvarig.
+      NotChanged: Ditt nya lösenord kan inte vara samma som ditt gamla lösenord
+    UsernameOrPassword:
+      Invalid: Användarnamn eller lösenord har felaktigt format
+    PasswordComplexityPolicy:
+      NotFound: Lösenordspolicy saknas
+      MinLength: Lösenordet är för kort
+      HasLower: Lösenordet måste innehålla små bokstäver
+      HasUpper: Lösenordet måste innehålla stora bokstäver
+      HasNumber: Lösenordet måste innehålla en siffra
+      HasSymbol: Lösenordet måste innehålla ett specialtecken
+    Code:
+      Expired: Koden är för gammal
+      Invalid: Koden är felaktig
+      Empty: Koden är tom
+      CryptoCodeNil: Cryptokoden är tom
+      NotFound: Koden hittades inte
+      GeneratorAlgNotSupported: Genereringsalgoritmen stöds inte
+    EmailVerify:
+      UserIDEmpty: UserID är tom
+    ExternalData:
+      CouldNotRead: Externa data kunde inte läsas rätt
+    MFA:
+      NoProviders: Det finns inga MFA-leverantörer tillgängliga
+      OTP:
+        AlreadyReady: Multifactor OTP (OneTimePassword) är redan konfigurerat
```

Comment:
Ibland "multifaktor" och ibland tvåfaktor, kan vara ok dock

Reply:
Frågan är om multifaktor ska skrivas med k istället för c här också btw

Reply:
Kollade upp lite. 

https://www.digg.se/digitala-tjanster/e-legitimering/tillitsnivaer-for-e-legitimering

Vi ska nog hålla det till tvåfaktor. "Multifaktor" verkar inte vara vedertaget.